assets/js/blocks/checkout/billing/index.js

```diff
@@ -7,8 +7,13 @@ import { InnerBlocks } from '@wordpress/editor';
 
 const getFieldBlock = ( field ) => {
 	const className = Array.isArray( field.class ) ? field.class.join( ' ' ) : null;
-	const { label, required } = field;
-	const attributes = { className, label, required };
+	const { label, required, visible } = field;
+	const attributes = { className, label, isRequired:required, isVisible:visible };
```

Comment:
Nitpicking here: :nerd_face: 

```suggestion
	const attributes = { className, label, isRequired: required, isVisible: visible };
```